pkg/asset/asset.go

```diff
@@ -101,8 +101,8 @@ func AncestryPath(ancestors []string) string {
 	return strings.Join(revAncestors, "/")
 }
 
-// UnwrapCAIResource will unwrap a K8S resource from the CAI payload and populate any omitted fields.
-func UnwrapCAIResource(asset map[string]interface{}) (*unstructured.Unstructured, error) {
+// ConvertCAIToK8s will convert a supported CAI Asset to a K8S resource and populate any omitted fields.
+func ConvertCAIToK8s(asset map[string]interface{}) (*unstructured.Unstructured, error) {
```

Comment:
`UnwrapCAIResource` -> `ConvertCAIToK8s` was born out of my confusion with the word "unwrap" when I first started working with this resource and because it let me eliminate references to `CAIResource` - but I would be fine with changing it back if that's preferred.

Reply:
I think this is clearer anyways, thanks!